apps/desktop/src/components/settings/views/sound.tsx

```
@@ -77,38 +79,124 @@ export default function Sound() {
     queryFn: () => listenerCommands.checkSystemAudioAccess(),
   });
 
+  const deviceQuery = useQuery<MicrophoneDeviceInfo>({
+    queryKey: ["microphoneDeviceInfo"],
+    queryFn: async () => {
+      try {
+        const result = await listenerCommands.getSelectedMicrophoneDevice();
+        return result;
+      } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)